datetime_tz/pytz_abbr.py

```diff
@@ -51,15 +51,51 @@
 class tzabbr(datetime.tzinfo):
   """A timezone abbreviation.
 
-  *WARNING*: This is not a tzinfo implementation! Trying to use this as tzinfo
-  object will result in failure.  We inherit from datetime.tzinfo so we can get
-  through the dateutil checks.
+  *WARNING*: This is a proxy object for an underlying `pytz` object only
+  intended to allow dateutil to probe the localized datetime for information.
+  datetime_tz's parsing logic will replace it with the relevant pytz zone
+  before returning to the end user.
   """
-  pass
+
+  def __init__(self, abbr, name, region, zone, dst):
```

Comment:
I assume no `super` call because this is a bit of an odd duck proxy?

Reply:
I just didn't think of it, actually. `datetime.tzinfo` doesn't define `__init__()`, so it currently makes no difference whether or not you call the superclass constructor, though maybe it's best to do so for reasons of future-proofing (though it would have to be a very odd set of circumstances indeed for this to make a difference, stranger things have happened).